app/templates/search/_briefs_results.html

```diff
@@ -1,50 +1,51 @@
-{% for brief in briefs %}
-<div class="search-result">
-    <h2 class="search-result-title">
-        <a class="govuk-link" href="{{ url_for('.get_brief_by_id', framework_family=framework_family, brief_id=brief.id) }}">{{ brief.title }}</a>
-    </h2>
+<ul>
+    {% for brief in briefs %}
+    <li class="app-search-result">
+        <h2 class="govuk-heading-s govuk-!-margin-bottom-1">
+            <a class="govuk-link" href="{{ url_for('.get_brief_by_id', framework_family=framework_family, brief_id=brief.id) }}">{{ brief.title }}</a>
+        </h2>
 
-    <ul class="search-result-important-metadata">
-        <li class="search-result-metadata-item">
-            {{ brief.organisation }}
-        </li>
-        <li class="search-result-metadata-item">
-            {{ brief.location }}
-        </li>
-    </ul>
-
-    <ul class="search-result-metadata">
-        <li class="search-result-metadata-item-inline">
-            {{ brief.lot.name }}
-        </li>
-        {% if 'specialistRole' in brief %}
-        <li class="search-result-metadata-item-inline">
-            {{ brief.specialistRole | capitalize() }}
-        </li>
-        {% endif %}
-    </ul>
-
-    <ul class="search-result-metadata">
-        {% if brief.status in outcomes.keys() %}
-            <li class="search-result-metadata-item">
-                Closed: {{ outcomes[brief.status] }}
+        <ul>
+            <li class="govuk-!-font-weight-bold">
+                <span class="govuk-visually-hidden">Organisation: </span>{{ brief.organisation }}
             </li>
-        {% else %}
-            <li class="search-result-metadata-item">
-                Published: {{ brief.publishedAt|dateformat }}
+            <li class="govuk-!-font-weight-bold">
+                <span class="govuk-visually-hidden">Location: </span>{{ brief.location }}
             </li>
-            <li class="search-result-metadata-item">
-                Deadline for asking questions: {{ brief.clarificationQuestionsClosedAt|dateformat }}
+        </ul>
+
+        <ul class="app-search-result__metadata">
+            <li class="govuk-!-display-inline-block">
+                {{ brief.lot.name }}
             </li>
-            <li class="search-result-metadata-item">
-                Closing: {{ brief.applicationsClosedAt|dateformat }}
+            {% if 'specialistRole' in brief %}
+            <li class="govuk-!-display-inline-block">
+                {{ brief.specialistRole | capitalize() }}
```

Comment:
Could we have a hyphen or something similar to textually separate the specialist role from the lot name?

Previously there was a larger space (which is probably not a perfect solution), and now it's hard to distinguish the two from each other visually:

#### Before

![Digital specialists       Programme manager](https://user-images.githubusercontent.com/503614/93449076-36862800-f8cc-11ea-9e44-fe4363d8cc05.png)

#### After

![Digital specialists Programme manager](https://user-images.githubusercontent.com/503614/93449119-456cda80-f8cc-11ea-82e3-13b4fb45f44c.png)